modules/vba_documents/app/workers/vba_documents/upload_remover.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+require 'vba_documents/object_store'
+
+module VBADocuments
+  class UploadRemover
+    include Sidekiq::Worker
+
+    EXPIRATION_TIME = 3.days
+
+    def perform
+      return unless Settings.vba_documents.s3.enabled
+      VBADocuments::UploadSubmission.where(
+        "status = 'received' AND s3_deleted IS NOT True and created_at < ?", EXPIRATION_TIME.ago
```

Comment:
Change first part to `status in ('received', 'processing', 'success', 'error')` - all the states that indicate either passed along to CMAPI or terminal error status.